source/api_documentation/network_integration/call_recording_condition/index.rst

```diff
@@ -0,0 +1,84 @@
+Call Recording Condition
+========================
+
+Manage call recording conditions for campaigns
+""""""""""""""""""""""""""""""""""""""""""""
+
+A call recording condition can be set for a campaign to determine whether or not the call should be recorded. The call will be recorded if the full condition is met.
+
+Set Call Recording Condition
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To conditionally record calls, set ``"record_calls"`` to ``true`` and set ``"call_recording_condition"`` to the expression you want to evaluate.
+
+POST
+
+``https://invoca.net/api/@@CAMPAIGN_FEATURES_API_VERSION/<network_id>/advertisers/<advertiser_id_from_network>/advertiser_campaigns/<advertiser_campaign_id_from_network>.json``
```

Comment:
I'm pretty sure this endpoint is only correct for api versions 2016-05-01 and below. I think versions above that do not require <advertiser_campaign_id_from_network> for POST requests

Reply:
These examples are updating existing campaigns, so it would need the campaign identifier. Would it be helpful to more explicitly state that these are updates and not creates?

Reply:
Ah, I see. In that case it might be helpful, but not totally necessary.